### ISSUE TITLE
Orientation error after taking photo using ImagePicker on iPhone devices

### ISSUE THREAD

**Issue by caigehui** [ant-design/ant-design-mobile#944]
#### Environment(required) | 环境（必填）

- antd-mobile version（antd-mobile版本）: 1.0.3
- Browser (or mark react-native) and its version（浏览器或react-native版本）: UC and safari
- Operating environment (e.g. OS name) and its version（操作系统版本或设备型号）: iOS 10.2

#### What did you do? Please provide steps to re-produce your problem.（请提供复现步骤）
Using ImagePicker just like the "Simple Image Picker" example  on official website
```
<ImagePicker
          files={files}
          onChange={this.onChange}
          onImageClick={(index, fs) => console.log(index, fs)}
          selectable={files.length < 5}
        />
```
It works fine on any other devices, but an image orientation error occurred after taking photo on iPhone 

#### What do you expected?（预期的正常效果）
taking photo with the right orientation

#### What happen?（发生了何种非正常现象）
Shooting a pen as following
![img_1221](https://cloud.githubusercontent.com/assets/9214907/23637746/67edd120-0318-11e7-83b7-2e7f387b3946.PNG)
Then the photo turn left for 90 degrees
![f06eb707b6875debce1f4219d38ac96b](https://cloud.githubusercontent.com/assets/9214907/23637728/52287070-0318-11e7-9206-9f708619d2e4.jpeg)

**Comment by paranoidjk** [comment]
iphone6s plus, safari, ios 10.2, all works fine, i can't reproduce you poblem, please provide a demo

http://codepen.io/paranoidjk/pen/LWpaKe

**Comment by caigehui** [comment]
@paranoidjk  
I still got the same problem when running the official demo on iPhone 6p/5s in any browser
## Reproduce:
1.Access https://mobile.ant.design/kitchen-sink/components/image-picker using iPhone
2.Took a photo in vertical direction
![img_1223](https://cloud.githubusercontent.com/assets/9214907/23641050/3b3cbc4a-032c-11e7-97a9-870e90f7ca21.PNG)
3.Then the pen became horizontal in thumb preview as following，rotating by 90 degrees, compared to the picture above 
![img_1224](https://cloud.githubusercontent.com/assets/9214907/23641052/3c1c7aa6-032c-11e7-8e9e-3925e0d62297.PNG)

**Comment by caigehui** [comment]
@paranoidjk 
Thanks for fixing the orientation problem, but the delete button is placed in the wrong position.
Take a look at the picture 
![a0faf43ae4946d1b2947ddb9bc69018b](https://cloud.githubusercontent.com/assets/9214907/23847088/bd798bd2-080b-11e7-9743-2bc6a87efcf0.jpeg)

**Comment by paranoidjk** [comment]
woops.... stupid mistake 😓  i will fix this in `1.0.6`

**Comment by paranoidjk** [comment]
@caigehui sorry for this bug. It had been fixed, please wait for `1.0.6` this friday. If you are in emergency, maybe you can temporarily overrider css to fix this problem.
